static/app/components/tooltip.tsx

```diff
@@ -318,11 +317,13 @@ export function DO_NOT_USE_TOOLTIP({
   );
 }
 
+interface ContainerProps {
+  containerDisplayMode?: React.CSSProperties['display'];
```

Comment:
But you're using `React.*` version of `CSSProperties` here 👀

Reply:
I'll consolidate to one, I need to cleanup a few extra things before we can merge this

Reply:
IMO (nit) prefer `React.*` for types, since it uses the global type namespace and reduces the number of imports